assets/icons/AddIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { SVGProps } from 'react'
+const AddIcon = ({ ...rest }: SVGProps<SVGSVGElement>) => {
```

Comment:
I don't think it's a better approach to create a simple SVG icon as a component. Can you please create `.svg` file and just simply import it? It'll work as a React component and will accept props (e.g. `className`).